kitsune/customercare/zendesk.py

```diff
@@ -42,6 +50,21 @@ def _user_to_zendesk_user(self, user, email):
             external_id=external_id,
         )
 
+    def get_user_by_email(self, email):
+        """Given an email, return a user from Zendesk."""
+        # This returns a generator, but we only want/expect one user
+        # If it returns more than one, we should fail
+        # Otherwise return the Zendesk user object
+        search_results = self.client.search(type="user", query=f"email:{email}")
+
+        user_found = None
+        for user in search_results:
+            if user_found is not None:
```

Comment:
Trying to think about this - I am checking if user_found exists, but you are saying to check if it doesn't exist. I think you might mean `if user_found` since it that would be the same as `if user_found is not None`. I'm going to leave it, as this this little section is hard to follow and I think this helps it be readable. I set user_found to None, explicitly, then I check if it is not None, explicitly.